Sources/TuistKit/Services/TestService.swift

```diff
@@ -113,36 +110,63 @@ final class TestService {
                     existing: testableSchemes.map(\.name)
                 )
             }
-            testSchemes = [scheme]
+            let testSchemes: [Scheme] = [scheme]
+            
+            try testSchemes.forEach { testScheme in
+                try self.testScheme(
```

Comment:
Little confused as to what's going on here, as it almost reads like this is just an unused variable

Reply:
true, I'll rewrite it a little to make it less confusing 👍